Source/WebCore/html/shadow/SwitchThumbElement.cpp

```diff
@@ -35,24 +37,16 @@ WTF_MAKE_ISO_ALLOCATED_IMPL(SwitchThumbElement);
 
 Ref<SwitchThumbElement> SwitchThumbElement::create(Document& document)
 {
-    return adoptRef(*new SwitchThumbElement(document));
+    Ref element = adoptRef(*new SwitchThumbElement(document));
+    ScriptDisallowedScope::EventAllowedScope eventAllowedScope { element };
+    element->setPseudo(ShadowPseudoIds::thumb());
+    return element;
+
 }
```

Comment:
It seems to me that we don't need SwitchThumbElement at all. Can it be replaced with a plain HTMLDivElement?

We generally don't want to add any new custom element subclasses unless they are elements with actual semantics.

Reply:
I filed https://bugs.webkit.org/show_bug.cgi?id=263968 to investigate this when we're closer to being done.